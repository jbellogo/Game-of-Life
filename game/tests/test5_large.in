new 10
init 
0 1 1 0 2 0 0 2
1 1 2 2 3 0 0 3
3 1 1 3 0 4 1 5
1 7 1 8 1 9 0 8
0 9 2 4 2 5 2 6
2 7 2 9 2 8 3 4
3 6 3 8 3 1 4 3 
4 5 4 7 4 9 4 1
5 3 5 4 5 6 5 8 
6 3 6 5 6 7 6 9
7 2 7 4 7 6 7 8
8 2 8 5 8 7 8 9
9 3 9 9 9 4
-1 -1

print 
step
print
step 
print
step 4
print
step 10
print
