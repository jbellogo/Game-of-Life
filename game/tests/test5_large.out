_XXX______
XX_XX_____
X_X____XX_
XX__XXX__X
X_XX_X_X_X
_XX_X_X_X_
__XX_X_X__
_XX_X_X_X_
XXXX_X_X__
XXX_X_X_XX
XX_XX_____
X___X_____
__X___XXX_
X___XX___X
X______X_X
________X_
________X_
X_______X_
_________X
X___XXXXX_
XX_XX_____
X_X_XX_X__
_X_XX_XXX_
_X___X___X
_________X
_______XXX
_______XXX
________XX
_____XX__X
_____XXXX_
XXXXXX____
X______XX_
XX_X___XX_
__X_XXXX_X
_________X
_______X__
__________
______X___
_____X___X
_____X_XX_
XXXXX_____
______XXX_
XXXXXX___X
_XXXXXXX_X
_____X_X__
__________
__________
__________
_____X_XX_
______X_X_
